.github/workflows/ci.yml

```diff
@@ -227,9 +227,32 @@ jobs:
       - name: Rename Binary with Target Name
         run: mv target/${{ matrix.target }}/release/tailcall target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
 
+      # Copy the CLI binary and rename it to include the name of the target platform
+      - name: Copy CLI binary
+        id: copy-cli
+        run: |
+          mkdir dist
+          cp target/${{ matrix.target }}/release/tailcall ./dist/tailcall-${{ matrix.target }}
```

Comment:
wouldn't it be easier to just copy to the destination folder inside `npm/*` and do not do copying thing in the `generate-packages` script again?